packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:meta/meta.dart';
+
+/// Signature for the callback passed to [compute].
+///
+/// {@macro flutter.foundation.compute.types}
+///
+/// Instances of [ComputeCallback] must be top-level functions, not closures or
+/// instance or static methods of classes.
```

Comment:
Why not static methods? AFAICT, isolates support them.

Reply:
Oh, cool. I may have misread the docs.